### PR TITLE
Feature GitHub workflow check

### PR DESCRIPTION
**Pull request by JamesABaker**
- md5 comparison if workflow folder is present.

### CODE REVIEW

File: Main_Loop.R
Comment (JamesABaker) [inline Main_Loop.R:1]: Shall we move this to the tests folder?

File: _scripts/gh.R [hunk 1]
```diff
@@ -24,9 +28,9 @@ gh_split_url <- function(url) {
 #' @param repo The repo to check in
 #' @param owner The owner of the repo
 #' @return TRUE if the file exists, FALSE if it does not
-gh_file_exists <- function(file, repo) {
+gh_file_exists <- function(file, repo, owner) {
```
Comment (JamesABaker) [inline _scripts/gh.R:31]: Owner was removed. I am not sure if this was deliberate but I have re-added it.

File: Main_Loop.R
Comment (JamesABaker) [reply Main_Loop.R:1]: Done.

**Comment by JamesABaker** [comment]
I suggest we squash this before merging. Most of the commits refer to fixing merge conflicts.

**Comment by Bisaloo** [comment]
Looking back at this, I don't think it's a very informative check. There are valid reasons for developers to tweak the workflow files. Indeed, even within Epiverse-TRACE, we have slight variations compared to the official `r-lib/actions` workflows. 

In all cases, if we wanted to restart this work, a new branch may be simpler given the importance of conflicts at this time.

**Pull request closed without merging**